.github/workflows/go.yml

```
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest", "windows-latest"]
-        go: ["1.22.x", "1.23.x"]
+        go: ["1.23.x", "1.24.x"]
```

Any reason to do not use stable and oldstable ?


```suggestion
        go: ["oldstabe", "stable"]
```